packages/shared/lib/auxiliary/popup/enums/popup-type.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum PopupType {
+    EnableLedgerBlindSigning = 'enableLedgerBlindSigning',
+}
```

Comment:
I like this approach, but we have to do this either for all popups or for none, because now this is the only one that follows this approach.

I would suggest cutting this code and pasting it to a new branch, since it would slow down this PR and create an issue for it under Software Quality milestone.

Reply:
My thinking was that we can do it incrementally, so if your changes touch a popup that isn't being used this way then you should try and change it. We don't have to have it for all or none - it's backwards compatible so long as we use the same string value for each popup type. I also think rather than `type`, `name` is a better descriptor, but that can be in a separate PR / task. 

FYI there already exists a backlog [task](https://github.com/iotaledger/firefly/issues/3974) to refactor popups :+1: